manifests/v/Viber/Viber/20.1.0.10/Viber.Viber.installer.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.2.6 $debug=AUSU.CRLF.5-1-19041-2673.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Viber.Viber
+PackageVersion: 20.1.0.10
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: user
```

Comment:
```suggestion
```
Added at the installer level. Didn't suggest a scope for WiX installer as it's affected by https://github.com/microsoft/winget-cli/issues/3011